kie-aries-blueprint/src/main/java/org/kie/aries/blueprint/namespace/KieObjectsInjector.java

```diff
@@ -180,12 +180,12 @@ private InternalKieModule createOsgiKModule(KieModuleModel kieProject) {
 
     protected void addKieModuleToRepo(KieModuleModel kieModuleModel) {
         String rootPath = configFilePath;
-        if ( rootPath.lastIndexOf( ':' ) > 0 ) {
+        if ( rootPath.lastIndexOf( ':' ) >= 2 ) {
```

Comment:
Do you mean that I could just add a comment or were you thinking to something else?